core/src/main/java/bio/terra/pearl/core/service/kit/KitRequestService.java

```diff
@@ -179,19 +182,22 @@ public void syncKitStatusesForStudy(Study study) {
                 Collectors.toMap(PepperKitStatus::getJuniperKitId, Function.identity(),
                         (kit1, kit2) -> !kit1.getCurrentStatus().equals("Deactivated") ? kit1 : kit2));
 
-        var studyEnvironments = studyEnvironmentService.findByStudy(study.getId());
-        for (StudyEnvironment studyEnvironment : studyEnvironments) {
-            var incompleteKits = findIncompleteKits(studyEnvironment.getId());
+        studyEnvironmentService.findByStudy(study.getShortcode(), environmentName).ifPresent(
+                studyEnvironment -> {
+                    var incompleteKits = findIncompleteKits(studyEnvironment.getId());
```

Comment:
Apparently we are good with using `var` rather than explicit types?

Reply:
I don't have a strong opinion on var -- I typically don't use `var` it unless the type is both particularly gruesome to type out and obvious.  Brian was more of a fan of it in regular use.  What's your take?